content/en/datadog_cloudcraft/_index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Cloudcraft in Datadog
+further_reading:
+- link: "https://www.datadoghq.com/blog/cloud-architecture-diagrams-cost-compliance-cloudcraft-datadog/"
+  tag: "Blog"
+  text: "Plan new architectures and track your cloud footprint with Cloudcraft (Standalone)"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Cloudcraft in Datadog is not supported on this site.</div>
+{{< /site-region >}}
+
+## Overview
+
+Cloudcraft offers a powerful, live read-only visualization tool for cloud architecture, enabling you to explore, analyze, and manage your infrastructure with ease. Not to be confused with the [Standalone Cloudcraft documentation][1], this guide outlines the functionality, setup, and use cases of Cloudcraft *in Datadog*, detailing its benefits for various user personas, and highlighting key features and capabilities.
+
+<div class="alert alert-info">This documentation applies to the Cloudcraft <em>in Datadog</em> product. For information on the standalone Cloudcraft product, please refer to the <a href="/cloudcraft"> Cloudcraft (Standalone) </a> documentation.</div>
```

Comment:
```suggestion
<div class="alert alert-info">This documentation applies to the Cloudcraft <em>in Datadog</em> product. For information on the standalone Cloudcraft product, please refer to the <a href="/cloudcraft">Cloudcraft (Standalone)</a> documentation.</div>
```
Some of this extra space is showing up in the link, which looks a bit odd, so I just removed any unnecessary spaces here